cloudify_cli/tests/commands/test_deployments.py

```diff
@@ -950,3 +952,305 @@ def test_deployment_modifications_get(self):
+        assert (call_args[1]['since'] - expected_since).total_seconds() < 0.1
+
+    def test_deployment_schedule_create_missing_since(self):
+        self.assertRaises(
+            ClickInvocationException,
+            self.invoke,
+            'cfy deployments schedule create backup -d dep1'
+        )
+
+    def test_deployment_schedule_create_missing_deployment_id(self):
+        self.assertRaises(
+            ClickInvocationException,
+            self.invoke,
+            'cfy deployments schedule create uninstall -s "12:33"'
+        )
+
+    def test_deployment_schedule_create_bad_time_expressions(self):
+        self.client.execution_schedules.create = MagicMock(
+            return_value=execution_schedules.ExecutionSchedule({}))
+        command = 'cfy deployments schedule create install -d dep1 -s {}'
+        error_msg = '{} is not a legal time format. accepted formats are ' \
+                    'YYYY-MM-DD HH:MM | HH:MM'
+
+        self.invoke(command.format('blah'),
+                    err_str_segment=error_msg.format('blah'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"15:33:18"'),
+                    err_str_segment=error_msg.format('15:33:18'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('99:99'),
+                    err_str_segment=error_msg.format('99:99'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"2000/1/1 09:17"'),
+                    err_str_segment=error_msg.format('2000/1/1 09:17'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"-1 min"'),
+                    err_str_segment=error_msg.format('-1 min'),
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"+10 dobosh"'),
+                    err_str_segment='10 dobosh is not a legal time delta',
+                    exception=CloudifyCliError)
+        self.invoke(command.format('"+rez"'),
+                    err_str_segment='+rez is not a legal time delta',
+                    exception=CloudifyCliError)
+
+    def test_deployment_schedule_create_bad_timezone(self):
+        self.invoke('cfy deployments schedule create install -d dep1 -s '
+                    '"7:15" --tz Mars/SpaceX',
+                    err_str_segment='Mars/SpaceX is not a recognized timezone',
+                    exception=CloudifyCliError)
+
+    def test_deployment_schedule_create_months_delta(self):
+        self.client.execution_schedules.create = MagicMock(
+            return_value=execution_schedules.ExecutionSchedule({}))
+        self.invoke('cfy deployments schedule create backup -d dep -s "+13mo"')
+        call_args = list(self.client.execution_schedules.create.call_args)
+
+        current_month = datetime.datetime.utcnow().month
+        current_year = datetime.datetime.utcnow().year
+        expected_month = 1 if current_month == 12 else current_month+1
```

Comment:
I wonder why flake didn't yell on `current_month+1`. It usually wants it as `current_month + 1`

Reply:
flake is flaky :)